components/bl-clock/README.md

```diff
@@ -13,6 +13,7 @@ This is a component for Backendless [UI-Builder](https://backendless.com/develop
 | Type: `type`                                 | Select [Clock: `clock`, Stopwatch: `stopwatch`, Timer: `timer`] | Clock: `clock`     |                          | NO           | YES        | Allows to determine the function of the component                                                                 |
 | Time Variant: `timeVariant`                  | Select [HH:MM:SS: `hhmmss`, HH:MM: `hhmm`, HH: `hh`]            | HH:MM:SS: `hhmmss` |                          | NO           | YES        | Allows to determine variant of displaying time                                                                    |
 | Stopwatch Scale: `stopwatchScale`            | Number                                                          | 3                  |                          | NO           | YES        | Allows to determine the scale of  stopwatch seconds                                                               |
+| Stopwatch Variant: `stopwatchVariant`        | Select [SS: `ss`, MM:SS: `mmss`, HH:MM:SS: `hhmmss`]            | SS: `ss`           |                          | NO           | YES        | Allows to determine variant of displaying stopwatch                                                               |
```

Comment:
I'm not sure if `SS` is something meaningful for nocoders. At least first google answer for `SS` that not what I'm interested for.

Reply:
I think it's the best name for time format. If googling "time format" everywhere is marked like this. And I think it's clear that if you're just Googling `SS`, that's probably not what you need